tools/cbmc/proofs/ARP/ARPGetCacheEntry/config_LLMNR/Makefile.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
AIUII, the Makefiles get generated by the CI every time it runs. If this is the case, please remove this file (in fact probably the whole directory) from the pull request.

Reply:
Current CI regenerates the makefiles from the makefile.json files.  Current CI does not generate the json files from the configuration files.  We can't do that until the configuration PRs are merged.

Reply:
@markrtuttle I see, but in that case, the configuration PR should also include a commit that deletes all the generated subdirectories. Or, we would need to introduce a commit later that deletes all the generated files.

That seems like unnecessary work to me, though. My preference would be that these PRs do not include generated files; furthermore, @mmuesly should change the titles of these PRs so that they end with `[depends: #594, #595]`. That way, we will not merge them until the configuration PRs have gone in.

If we merge the proof PRs in before the configuration PRs, then there is actually no guarantee that the configuration-generation script actually works! Therefore, I advocate merging #594 and #595, and then re-triggering the CI on all of the proof PRs.

Reply:
I think it would be easier on the reviewers if the commits did not include any of the generated files (and that goes for even the stuff we generate from Makefile.json).    But if any customers want to replay the proofs, they would probably benefit from having the generated files in the commit (one less thing for them to do).